util/proxy.go

```diff
@@ -0,0 +1,49 @@
+package util
+
+import (
+	"context"
+	"net"
+	"net/url"
+	"sync"
+
+	"github.com/go-sql-driver/mysql"
+	netproxy "golang.org/x/net/proxy"
+
+	"github.com/pingcap/tipocket/pkg/test-infra/fixture"
+)
+
+var (
+	setMySQLProxyOnce sync.Once
+)
+
+// SetMySQLProxy sets the proxy mysql dial specified in the command flag,
+// and makes underlying connections directly.
+func SetMySQLProxy() {
+	if fixture.Context.K8sProxy == "" {
+		return
+	}
+	dialer := fromURL(fixture.Context.K8sProxy)
```

Comment:
How about renaming the flag `K8sProxy` to `MySQLProxy`?

Reply:
Looks good to me.